src/api/article/controller.js

```diff
@@ -53,14 +58,25 @@ export const create = async ({ body }, res, next) => {
     const articleBrandQuery = ArticleBrand.findOne({ name: body.brand });
     const articleBrand = (await articleBrandQuery.exec()) || new ArticleBrand({ name: body.brand });
     await articleBrand.save();
-    const articleBody = body;
-    articleBody.brand = articleBrand._id; // eslint-disable-line no-underscore-dangle
+
+    const articleCollection = await ArticleCollection.findOne({ slug: body.collectionSlug });
+
+    const articleBody = omit(body, ['collectionSlug']);
```

Comment:
is it added to model `collectionSlug` if not omit it here?

Reply:
if not, that code looks more readable:

Reply:
1. `omit` here is redundant although we may want to keep it to emphasize `collectionSlug` is not a part of a model.

2. Your approach raises a `property _id of undefined` error.
The code below works:
const articleBody = {
      ...body,
      brand: articleBrand._id,
      collectionId: articleCollection ? articleCollection._id : null,
    };
Although `null` is not the same as nothing: `getAll` or `getOne` than return a `collectionId` property with `null` value while for other objects `collectionId` is not returned at all.

Reply:
better: ```jscollectionId: articleCollection && articleCollection._id```